### PR TITLE
ClippingPlanesCollection -> ClippingPlaneCollection

### PR DESCRIPTION
**Pull request by hpinkos**
I think it makes more sense to say `ClippingPlaneCollection` instead of `ClippingPlanesCollection`
We say `BillboardCollection` not `BillboardsCollection`.

I have some other questions and comments related to #6026, I'm going to make them in that PR

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Signed CLA is on file.

@hpinkos, thanks for the pull request! Maintainers, we have a signed CLA from @hpinkos, so you can review this at any time.

:warning: I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

File: Source/Scene/Instanced3DModel3DTileContent.js [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
+    '../Core/RequestType',
+    '../Core/RuntimeError',
+    '../Core/Transforms',
+    '../Core/TranslationRotationScale',
+    '../Renderer/Pass',
+    './Cesium3DTileBatchTable',
+    './Cesium3DTileFeature',
+    './Cesium3DTileFeatureTable',
+    './ClippingPlaneCollection',
+    './ModelInstanceCollection'
```
Comment (ggetz) [inline Source/Scene/Instanced3DModel3DTileContent.js]: I think these should be indented

**Comment by ggetz** [comment]
Thanks @hpinkos, looks good other than that one formatting comment.

**Comment by hpinkos** [comment]
Thanks @ggetz!

**Pull request merged**